third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -544,37 +545,31 @@ settings.backup_configuration.binary_log_enabled are both set to true.`,
 			"server_ca_cert": {
 				Type:     schema.TypeList,
 				Computed: true,
-				MaxItems: 1,
```

Comment:
I'm assuming the SDK newly validates this case?

Reply:
I don't remember making this change either - I think @paddycarver may have more context here as well. If not, I'll have to dig into it a bit more.

Reply:
Yes, this is part of a new SDK validation; as users can't input _any_ values for this, you can't have validations like MaxItems around what values users are allowed to input. Same with the AtLeastOneOfs below; users can't provide _any_ of them.

Reply:
🎉  great to hear!